components/hab/src/main.rs

```diff
@@ -901,9 +906,37 @@ fn sub_pkg_bulkupload(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
                       &artifact_path.display(),
                       key_path.display()))?;
     ui.status(Status::Found,
-              format!("{} artifacts for upload.", artifact_paths.len()))?;
-
-    for artifact_path in artifact_paths {
+              format!("{} artifact(s) for upload.", artifact_paths.len()))?;
+
+    if create_origins {
+        let mut origins: Vec<String> = Vec::new();
+        ui.status(Status::Discovering,
+                  "origin names from artifact metadata..".to_string())?;
+        for artifact_path in &artifact_paths {
+            let ident = PackageArchive::new(&artifact_path).ident()?;
+            origins.push(ident.origin);
+        }
+        origins.sort();
+        origins.dedup();
+        for origin in &origins {
+            ui.status(Status::Custom(Glyph::CheckMark, "".to_string()), origin)?;
+        }
+        ui.status(Status::Found,
+                  format!("{} origin(s) that may need to be created.", origins.len()))?;
+        if !origins.is_empty() && !accept {
+            ui.warn("Your Builder id will own any origin created. Be sure this is what you \
+                     intend!\nOwnership transfer is not yet implemented and will require SQL \
```

Comment:
This message seems a bit heavy to me.  We should fast follow with the origin ownership transfer API/cli so we dont have to put in draconian warnings that will live around for a long time and be incorrect as soon as that API lands.

Reply:
This Builder issue exists for ownership transfer https://github.com/habitat-sh/builder/issues/36